verification/curator-service/src/model/source.ts

```diff
@@ -15,17 +15,68 @@ const originSchema = new mongoose.Schema({
     },
 });
 
+interface Field {
+    name: string;
+    regexp: string;
+}
+
+const fieldSchema = new mongoose.Schema({
+    url: {
+        type: String,
+        required: 'Enter a dotted.field.name',
+    },
+    regexp: {
+        type: String,
+        required: 'Enter a field extraction regexp',
+    },
+});
+
+interface Parsing {
+    fields: Array<Field>;
+}
+
+const parsingSchema = new mongoose.Schema({
+    fields: {
+        type: [fieldSchema],
+    },
+});
+
+interface Automation {
+    name: string;
+    tag: string;
+    active: boolean;
+    scheduleExpression: string;
+    parsing: Parsing;
+}
+
+const automationSchema = new mongoose.Schema({
+    name: {
+        name: {
+            type: String,
+            required: 'Enter an automation rule name',
```

Comment:
Re: required: this is the "name of the image in docker registry" -- what if they're using the `parsing` field, instead?